framework/test/unit/modules/reward/reward_module.spec.ts

```diff
@@ -46,4 +48,39 @@ describe('RewardModule', () => {
 			expect(rewardModule['_moduleConfig']).toEqual(moduleConfig);
 		});
 	});
+
+	const { brackets, offset, distance } = moduleConfig as {
+		brackets: ReadonlyArray<bigint>;
+		offset: number;
+		distance: number;
+	};
+
+	describe.each(Object.entries(brackets))(
+		'afterBlockExecute test for brackets',
```

Comment:
i don't think it's necessary to test all the brackets assuming the util is tested well

Reply:
Then you think I should test the just first bracket in a single test?

Reply:
yes, i think just first bracket after offset should be good

Reply:
done